ide/coqide/coqide.ml

```diff
@@ -1533,16 +1533,18 @@ let build_ui () =
     ("Forward", "_Forward", `GO_DOWN, Nav.forward_one, "Forward one command", "Down");
     ("Backward", "_Backward", `GO_UP, Nav.backward_one, "Backward one command", "Up");
     ("Run to cursor", "Run to _cursor", `JUMP_TO, Nav.run_to_cursor, "Run to cursor", "Right");
+    ("Reset", "_Reset", `GOTO_TOP, Nav.restart, "Reset Coq", "Home");
     ("Run to end", "_Run to end", `GOTO_BOTTOM, Nav.run_to_end, "Run to end", "End");
+    ("Fully check", "_Fully check", `EXECUTE, Nav.join_document, "Fully check the document", "Return");
```

Comment:
Yes, “Return” is another name for “Enter”. I think I used it because GTK didn’t recognize “Enter”. It is bound to Fully check with the modifier for navigation, so it gives Alt-Enter. Maybe it is a bad choice.